app/views/gigs/manage.html.slim

```diff
@@ -0,0 +1,20 @@
+= provide :title, "Manage Gigs"
+section.main-content
+  .row
+    .two.columns.offset-by-one.schedule
+      article
+        = link_to "Add gig", new_gig_path
+
+  - Gig.all_ascending.each do |gig|
+    -if gig.date + gig.days >= Date.today
```

Comment:
Even better, make the expired class that was the only difference between these blocks conditionally inserted, and use a unified set otherwise.